dll/win32/shell32/lang/ro-RO.rc

```diff
@@ -136,14 +137,14 @@ CAPTION "Mesaj"
 FONT 8, "MS Shell Dlg"
 BEGIN
     DEFPUSHBUTTON "&Da", IDYES, 34, 69, 53, 14, WS_GROUP | WS_TABSTOP
-    PUSHBUTTON "D&a pentru tot", IDC_YESTOALL, 92, 69, 65, 14
-    PUSHBUTTON "&Nu", IDNO, 162, 69, 53, 14
-    PUSHBUTTON "Revocare", IDCANCEL, 220, 69, 53, 14
+    PUSHBUTTON "D&a pentru tot", IDC_YESTOALL, 92, 69, 65, 14, WS_TABSTOP
```

Comment:
https://learn.microsoft.com/en-us/windows/win32/menurc/pushbutton-control
So the extra `WS_TABSTOP` you've added are unnecessary; please remove them.

Reply:
Done!